doc/source/guide/geometry_guide.rst

```diff
@@ -0,0 +1,240 @@
+-------------------------------------
+The initial state of the system corresponds to motion parameters :math:`m = 0`.
+To determine the spatial position :math:`p(m, u)` of a detector point at a given configuration, both motion parameter :math:`m` and detector parameter :math:`u` need to be specified.
+
+The vector pointing from the origin to a detector point is decomposed into two components:
+
+- a detector reference point :math:`r = r(m)` only depending on the motion parameter (`Geometry.det_refpoint`),
+- a shift :math:`s = s(u)` within the detector only depending on the detector parameter (`Detector.surface`).
+
+The total displacement is given by
+
+    .. math::
+        p(m, u) = r(m) + s(u)
+
+Here, the shift in the detector is defined in the reference coordinate system, i.e., as an intrinsic shift.
+
+In the 2d parallel beam example, :math:`r(m)` corresponds to :math:`\theta(\varphi)`.
+The intrinsic shift can be defined in the reference state of the detector aligned with the :math:`y` axis.
+In this case, :math:`s(u) = (0, u)`.
+Both the detector reference point and the extrinsic shift in the detector can be expressed as rotation of vectors in an initial state.
+Here, this initial state can be defined as :math:`\theta(0) = (1, 0)` and :math:`s(u) = (0, u)`, and the detector point at :math:`\varphi, u` is given by
+
+    .. math::
+        & p(\varphi, u) = R(\varphi)
+        \begin{pmatrix}
+            1 \\
+            0
+        \end{pmatrix}
+        + R(\varphi)
+        \begin{pmatrix}
+            0 \\
+            u
+        \end{pmatrix},
+
+        & R(\varphi) =
+        \begin{pmatrix}
+            \cos\varphi & -\sin\varphi \\
+            \sin\varphi & \cos\varphi
+        \end{pmatrix}
+
+The rotation matrix :math:`R(\varphi)` is exposed as `Geometry.rotation_matrix`.
+
+Determining the initial configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In two dimensions, the default initial configuration of geometries in ODL is
+
+    .. math::
+        r(0) = \alpha
+        \begin{pmatrix}
+            1 \\
+            0
+        \end{pmatrix},
+        \ s(u) = u
+        \begin{pmatrix}
+            0 \\
+            1
+        \end{pmatrix}.
+
+If a different initial detector position :math:`r(0) = \alpha (\cos\psi, \sin\psi)^{\mathrm{T}}` is chosen, the initial detector axis is taken to be :math:`s(1) = (-\sin\psi, \cos\psi)` by default.
+
+In three dimensions, there is no unique way to rotate one vector to another, which is why a convection is required in this case.
+The standard configuration in 3d is
+
+    .. math::
+        r(0) =
+        \begin{pmatrix}
+            -1 \\
+            0  \\
+            0
+        \end{pmatrix},
+        \
+        s(1, 0) =
+        \begin{pmatrix}
+            0 \\
+            1  \\
+            0
+        \end{pmatrix},
+        \
+        s(0, 1) =
+        \begin{pmatrix}
+            0 \\
+            0  \\
+            1
+        \end{pmatrix}
```

Comment:
Also here, things should be general (i.e. use `s_0` and `s_1` as basis vectors).